packages/opentelemetry-instrumentation-bedrock/opentelemetry/instrumentation/bedrock/__init__.py

```diff
@@ -62,36 +89,72 @@ def _set_span_attribute(span, name, value):
 def _with_tracer_wrapper(func):
     """Helper for providing tracer for wrapper functions."""
 
-    def _with_tracer(tracer, to_wrap):
+    def _with_tracer(
+        tracer,
+        metricParams,
```

Comment:
If you are adding a new parameter, I prefer put it as last one.

Reply:
I just want to put the metric parameters near by tracer and it's an internal function only.